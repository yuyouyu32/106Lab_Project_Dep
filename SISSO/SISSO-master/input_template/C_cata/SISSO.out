Version SISSO.3.0.2, June, 2020.
================================================================================

Reading data from train.dat ...
Reading parameters from SISSO.in: 
--------------------------------------------------------------------------------
Restarts :     F
Descriptor dimension:        4
Property type:     1
Total number of properties:        1
Task_weighting:        1
Number of samples for each property:      112
Number of scalar features:       18
Number of recursive calls for feature transformation (rung of the feature space):        3
Max feature complexity (number of operators in a feature):        3
Number of dimension(unit)-type (for dimension analysis):       18
Dimension type for each primary feature: 
  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
Lower bound of the max abs. data value for the selected features:     0.10000E-02
Upper bound of the max abs. data value for the selected features:     0.10000E+06
Size of the SIS-selected (single) subspace :     1000    1000    1000    1000
Operators for feature construction: (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) 
Method for sparsification:  L0        
Number of the top ranked models to output:      100
Fitting intercept?      T
Metric for model selection: RMSE
--------------------------------------------------------------------------------

Feature Construction and Descriptor Identification (FCDI) starts ...

iteration:   1
--------------------------------------------------------------------------------
FC starts ...
Standard Deviation (SD) of property 001:    1.25592
Total number of features in the space phi00:             18
Total number of features in the space phi01:            639
Total number of features in the space phi02:         185552
Total number of features in the space phi03:        1314474
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:           93.17
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       1000
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  1D descriptor (model): 
Total RMSE,MaxAE:   0.365296  0.809140
@@@descriptor: 
                     19:[(epsilon_d_eV/sin((r_d_Ang/PE)))]
       coefficients_001:    -0.6448134677E+00
          Intercept_001:    -0.5815422645E+00
         RMSE,MaxAE_001:     0.3652962029E+00    0.8091401991E+00
================================================================================
Wall-clock time (second) for this DI:            4.97
DI done!

iteration:   2
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            639
Total number of features in the space phi02:         185552
Total number of features in the space phi03:        1314474
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:           92.43
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       2000
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  2D descriptor (model): 
Total RMSE,MaxAE:   0.233142  0.748353
@@@descriptor: 
                     20:[(PE/sin((r_d_Ang/epsilon_d_eV)))]
                   1050:[((CN/(W_eV)^2)/site_no)]
       coefficients_001:    -0.6442553127E+00    0.1543792457E+02
          Intercept_001:    -0.2166845173E+01
         RMSE,MaxAE_001:     0.2331422067E+00    0.7483529957E+00
================================================================================
Wall-clock time (second) for this DI:            5.47
DI done!

iteration:   3
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            639
Total number of features in the space phi02:         185552
Total number of features in the space phi03:        1314474
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:           91.07
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       3000
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  3D descriptor (model): 
Total RMSE,MaxAE:   0.190062  0.438996
@@@descriptor: 
                     37:[(((PE*epsilon_d_eV)/W_eV)/r_d_Ang)]
                   1439:[((sin(site_no)/W_d_eV)/epsilon_d_eV)]
                   2061:[sin(((f_d*f_sp)/W_d_eV))]
       coefficients_001:    -0.2986841814E+01   -0.1954611778E+01    0.2936925958E+00
          Intercept_001:     0.2918886725E-02
         RMSE,MaxAE_001:     0.1900621839E+00    0.4389958731E+00
================================================================================
Wall-clock time (second) for this DI:           24.18
DI done!

iteration:   4
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            639
Total number of features in the space phi02:         185552
Total number of features in the space phi03:        1314474
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:           97.43
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       4000
L0 starts ...

Final model/descriptor to report
================================================================================
  4D descriptor (model): 
Total RMSE,MaxAE:   0.167187  0.515095
@@@descriptor: 
                     37:[(((PE*epsilon_d_eV)/W_eV)/r_d_Ang)]
                   1439:[((sin(site_no)/W_d_eV)/epsilon_d_eV)]
                   2061:[sin(((f_d*f_sp)/W_d_eV))]
                   3002:[(exp(V_ad^2)/cos(CN))]
       coefficients_001:    -0.2991290435E+01   -0.2168928626E+01    0.2980923470E+00   -0.7790753113E-03
          Intercept_001:     0.2732056790E-02
         RMSE,MaxAE_001:     0.1671869735E+00    0.5150954377E+00
================================================================================
Wall-clock time (second) for this DI:         5987.81
DI done!

FCDI done !

--------------------------------------------------------------------------------
Total wall-clock time (second):         6398.06

                                               Have a nice day !    

