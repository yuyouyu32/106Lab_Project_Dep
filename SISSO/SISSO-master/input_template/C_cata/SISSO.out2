Version SISSO.3.0.2, June, 2020.
================================================================================

Reading data from train.dat ...
Reading parameters from SISSO.in: 
--------------------------------------------------------------------------------
Restarts :     T
Descriptor dimension:        4
Property type:     1
Total number of properties:        1
Task_weighting:        1
Number of samples for each property:      112
Number of scalar features:       18
Number of recursive calls for feature transformation (rung of the feature space):        3
Max feature complexity (number of operators in a feature):        3
Number of dimension(unit)-type (for dimension analysis):       18
Dimension type for each primary feature: 
  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
Lower bound of the max abs. data value for the selected features:     0.10000E-02
Upper bound of the max abs. data value for the selected features:     0.10000E+06
Size of the SIS-selected (single) subspace :      100     100     100     100
Operators for feature construction: (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) 
Method for sparsification:  L0        
Number of the top ranked models to output:      100
Fitting intercept?      T
Metric for model selection: RMSE
--------------------------------------------------------------------------------

Feature Construction and Descriptor Identification (FCDI) starts ...

iteration:   4
--------------------------------------------------------------------------------

DI starts ...
total number of SIS-selected features from all iterations:       1600
