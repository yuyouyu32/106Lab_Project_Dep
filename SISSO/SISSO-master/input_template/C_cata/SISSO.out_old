Version SISSO.3.0.2, June, 2020.
================================================================================

Reading data from train.dat ...
Reading parameters from SISSO.in: 
--------------------------------------------------------------------------------
Restarts :     F
Descriptor dimension:        4
Property type:     1
Total number of properties:        1
Task_weighting:        1
Number of samples for each property:      112
Number of scalar features:       18
Number of recursive calls for feature transformation (rung of the feature space):        3
Max feature complexity (number of operators in a feature):        3
Number of dimension(unit)-type (for dimension analysis):       18
Dimension type for each primary feature: 
  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
Lower bound of the max abs. data value for the selected features:     0.10000E-02
Upper bound of the max abs. data value for the selected features:     0.10000E+06
Size of the SIS-selected (single) subspace :      400     400     400     400
Operators for feature construction: (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(exp)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) 
Method for sparsification:  L0        
Number of the top ranked models to output:      100
Fitting intercept?      T
Metric for model selection: RMSE
--------------------------------------------------------------------------------

Feature Construction and Descriptor Identification (FCDI) starts ...

iteration:   1
--------------------------------------------------------------------------------
FC starts ...
Standard Deviation (SD) of property 001:    1.25592
Total number of features in the space phi00:             18
Total number of features in the space phi01:            297
Total number of features in the space phi02:          37181
Total number of features in the space phi03:         159269
Size of the SIS-selected subspace from phi03:        400
Wall-clock time (second) for this FC:            4.55
FC done!

DI starts ...
total number of SIS-selected features from all iterations:        400
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  1D descriptor (model): 
Total RMSE,MaxAE:   0.357594  1.340820
@@@descriptor: 
                      1:[cbrt((site_no*sin(DOS_d_eV^(-1))))]
       coefficients_001:    -0.3413108231E+01
          Intercept_001:     0.6766700366E+01
         RMSE,MaxAE_001:     0.3575937907E+00    0.1340819514E+01
================================================================================
Wall-clock time (second) for this DI:            0.03
DI done!

iteration:   2
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            297
Total number of features in the space phi02:          37181
Total number of features in the space phi03:         159269
Size of the SIS-selected subspace from phi03:        400
Wall-clock time (second) for this FC:            4.69
FC done!

DI starts ...
total number of SIS-selected features from all iterations:        800
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  2D descriptor (model): 
Total RMSE,MaxAE:   0.238944  0.660675
@@@descriptor: 
                     82:[cbrt((site_no*cbrt(DOS_d_eV^(-1))))]
                     96:[((CN*(f_d)^3)*epsilon_d_eV)]
       coefficients_001:    -0.3125200042E+01   -0.8048178788E-04
          Intercept_001:     0.6090308581E+01
         RMSE,MaxAE_001:     0.2389436647E+00    0.6606746903E+00
================================================================================
Wall-clock time (second) for this DI:            1.10
DI done!

iteration:   3
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            297
Total number of features in the space phi02:          37181
Total number of features in the space phi03:         159269
Size of the SIS-selected subspace from phi03:        400
Wall-clock time (second) for this FC:            4.40
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       1200
L0 starts ...

Model/descriptor for generating residual:
================================================================================
  3D descriptor (model): 
Total RMSE,MaxAE:   0.210862  0.676102
@@@descriptor: 
                     82:[cbrt((site_no*cbrt(DOS_d_eV^(-1))))]
                    157:[((CN*(f_d)^2)*epsilon_d_eV)]
                    803:[abs(cos(IP_eV)-sin(site_no))]
       coefficients_001:    -0.3363502785E+01   -0.7843674581E-03   -0.2560028777E+00
          Intercept_001:     0.6516846105E+01
         RMSE,MaxAE_001:     0.2108624079E+00    0.6761019341E+00
================================================================================
Wall-clock time (second) for this DI:         1265.36
DI done!

iteration:   4
--------------------------------------------------------------------------------
FC starts ...
File containing the features to be rejected: feature_space/Uspace.name
Total number of features in the space phi00:             18
Total number of features in the space phi01:            297
Total number of features in the space phi02:          37181
Total number of features in the space phi03:         159269
Size of the SIS-selected subspace from phi03:        400
Wall-clock time (second) for this FC:            4.83
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       1600
