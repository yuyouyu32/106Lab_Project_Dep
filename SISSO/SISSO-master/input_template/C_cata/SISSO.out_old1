Version SISSO.3.0.2, June, 2020.
================================================================================

Reading data from train.dat ...
Reading parameters from SISSO.in: 
--------------------------------------------------------------------------------
Restarts :     F
Descriptor dimension:        5
Property type:     1
Total number of properties:        1
Task_weighting:        1
Number of samples for each property:      112
Number of scalar features:       18
Number of recursive calls for feature transformation (rung of the feature space):        3
Max feature complexity (number of operators in a feature):        3
Number of dimension(unit)-type (for dimension analysis):       18
Dimension type for each primary feature: 
  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
Lower bound of the max abs. data value for the selected features:     0.10000E-02
Upper bound of the max abs. data value for the selected features:     0.10000E+06
Size of the SIS-selected (single) subspace :     1000    1000    1000    1000    1000
Operators for feature construction: (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(sqrt)(cbrt)(log)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(sqrt)(cbrt)(log)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(sqrt)(cbrt)(log)(|-|)(^6)(sin)(cos) 
Method for sparsification:  L0        
Number of the top ranked models to output:      100
Fitting intercept?      T
Metric for model selection: RMSE
--------------------------------------------------------------------------------

Feature Construction and Descriptor Identification (FCDI) starts ...

iteration:   1
--------------------------------------------------------------------------------
FC starts ...
Standard Deviation (SD) of property 001:    1.25592
Total number of features in the space phi00:             18
Total number of features in the space phi01:            673
Total number of features in the space phi02:         228352
Total number of features in the space phi03:        1577604
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:          155.77
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       1000
