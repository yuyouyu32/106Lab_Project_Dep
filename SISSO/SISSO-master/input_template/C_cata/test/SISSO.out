Version SISSO.3.0.2, June, 2020.
================================================================================

Reading data from train.dat ...
Reading parameters from SISSO.in: 
--------------------------------------------------------------------------------
Restarts :     F
Descriptor dimension:        1
Property type:     1
Total number of properties:        1
Task_weighting:        1
Number of samples for each property:      112
Number of scalar features:       18
Number of recursive calls for feature transformation (rung of the feature space):        3
Max feature complexity (number of operators in a feature):        3
Number of dimension(unit)-type (for dimension analysis):       18
Dimension type for each primary feature: 
  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
Lower bound of the max abs. data value for the selected features:     0.10000E-02
Upper bound of the max abs. data value for the selected features:     0.10000E+06
Size of the SIS-selected (single) subspace :     1000
Operators for feature construction: (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) (+)(-)(*)(/)(exp)(exp-)(^-1)(^2)(^3)(cbrt)(|-|)(^6)(sin)(cos) 
Method for sparsification:  L0        
Number of the top ranked models to output:      100
Fitting intercept?      T
Metric for model selection: RMSE
--------------------------------------------------------------------------------

Feature Construction and Descriptor Identification (FCDI) starts ...

iteration:   1
--------------------------------------------------------------------------------
FC starts ...
Standard Deviation (SD) of property 001:    1.25592
Total number of features in the space phi00:             18
Total number of features in the space phi01:            639
Total number of features in the space phi02:         185552
Total number of features in the space phi03:        1314474
Size of the SIS-selected subspace from phi03:       1000
Wall-clock time (second) for this FC:          106.29
FC done!

DI starts ...
total number of SIS-selected features from all iterations:       1000
L0 starts ...

Final model/descriptor to report
================================================================================
  1D descriptor (model): 
Total RMSE,MaxAE:   0.349358  1.008018
@@@descriptor: 
                      1:[(W_eV/cbrt((site_no*DOS_d_eV^(-1))))]
       coefficients_001:     0.5850917853E+00
          Intercept_001:     0.7026133504E-01
         RMSE,MaxAE_001:     0.3493579951E+00    0.1008017759E+01
================================================================================
Wall-clock time (second) for this DI:            0.06
DI done!

FCDI done !

--------------------------------------------------------------------------------
Total wall-clock time (second):          106.72

                                               Have a nice day !    

